### PR TITLE
Automated cherry pick of #43493

### PR DESCRIPTION
**Pull request by pipejakob**
Cherry pick of #43493 on release-1.6.

#43493: Better messaging when GKE certificate signing fails.

### CODE REVIEW

**Comment by pipejakob** [comment]
/release-note Better messaging when GKE certificate signing fails.

**Comment by pipejakob** [comment]
Ran `./hack/update-openapi-spec.sh` to appease the verify step.

File: api/openapi-spec/swagger.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.6.0"
+   "version": "v1.6.1"
```
Comment (roberthbailey) [inline api/openapi-spec/swagger.json]: why is this part of your PR?

Comment (pipejakob) [inline api/openapi-spec/swagger.json]: The Jenkins verification on this PR failed due to `hack/verify-federation-openapi-spec.sh`, so I ran `./hack/update-openapi-spec.sh` and amended the commit. I'm not sure if the verify failure would have sorted itself out without any intervention, but I can pull that change back out so it's purely a cherry-pick of the master commit.

This is my first cherry-pick, so apologies if I've bungled the process.

**Comment by pipejakob** [comment]
I have reverted the `swagger.json` change, so this diff is back to purely cherry picking the parent PR.

**Comment by pipejakob** [comment]
@k8s-bot gci gce e2e test this

**Comment by mikedanese** [comment]
@k8s-bot verification test this

**Comment by pipejakob** [comment]
@k8s-bot verify test this

**Comment by pipejakob** [comment]
@k8s-bot kops aws e2e test this

**Pull request merged**